src/font-stylesheet-timeout.js

```diff
@@ -89,7 +89,10 @@ function maybeTimeoutFonts(win) {
       // To avoid blocking the render, we assign a non-matching media
       // attribute first…
       const media = link.media || 'all';
-      link.media = 'not-matching';
+      // Move from generic 'non-matching' media to 'print', avoiding
+      // issue where Chromium based browsers will avoid downloading
+      // these media assets aggressively. (https://bugs.chromium.org/p/chromium/issues/detail?id=977573)
```

Comment:
I don't think this comment is necessary (they can review the git-blame for context) as long as the reasoning comment (2 lines above) is kept.